server/app/views/components/Icons.java

```diff
@@ -44,6 +44,17 @@ public enum Icons {
       "M7 8h10M7 12h4m1 8l-4-4H5a2 2 0 01-2-2V6a2 2 0 012-2h14a2 2 0 012 2v8a2 2 0 01-2 2h-3l-4"
           + " 4z"),
   ARROW_LEFT(24, "m12 20-8-8 8-8 1.425 1.4-5.6 5.6H20v2H7.825l5.6 5.6Z"),
+  CAKE(
+      24,
+      "M12 7C13.11 7 14 6.1 14 5C14 4.62 13.9 4.27 13.71 3.97L12"
```

Comment:
Melanie, have we tested at 200% zoom? Sometime icon behaves differently on zoom in/zoom out. Similarly, can we test it out in mobile view too? Sorry for the added work.

Reply:
It looks good at 200% zoom.  The icon also looks fine in mobile view, although the cards themselves don't.  I was under the impression that we didn't support mobile view currently on the admin side.

Reply:
Thanks for testing. My thought is if folks use this icon again in some other page, we should be able to support it.